plugins/quickbooks-online/src/src/Facade/QuickBooksFacade.php

```diff
@@ -447,11 +458,38 @@ private function getAccounts(): array
                 $activeAccounts[$account->Id] = $account;
             }
 
-            return $activeAccounts;
         } catch (\Exception $exception) {
             $this->logger->error(
                 sprintf('Account: Getting all Accounts failed with error %s.', $exception->getMessage())
             );
         }
+
+        return $activeAccounts;
+    }
+
+    /**
+     * @throws QBAuthorizationException
+     * @throws \ReflectionException
+     */
+    private function invalidateTokens(): void
+    {
+        $this->logger->info(
+            'Connection failed. You need confirm your connection again.'
```

Comment:
Connection failed. You need `to` confirm your connection again. `Follow the initial steps`